jenkins/scripts/nvdf.py

```
@@ -0,0 +1,248 @@
+        json_data = json.dumps(json_data)
+        if DISABLE_NVDF_FOR_LOCAL_TEST:
+            # NVDF.logger.info("NVDF is disabled, skip posting to NVDataFlow: {}".format(json_data))
+            return
+        url = "{}/dataflow2/{}/posting".format(NVDF_BASE_URL, project)
+        headers = {
+            "Content-Type": "application/json",
+            "Accept-Charset": "UTF-8"
+        }
+        retry_time = 5
+        while retry_time:
+            res = requests.post(
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)